src/components/refactor/dialogs/voltage-level-creation/bus-bar-section/connectivity.js

```diff
@@ -0,0 +1,72 @@
+import { gridItem } from 'components/dialogs/dialogUtils';
+import {
+    BUS_BAR_SECTIONS,
+    FROM_BBS,
+    ID,
+    SWITCH_KIND,
+    TO_BBS,
+} from 'components/refactor/utils/field-constants';
+import AutocompleteInput from 'components/refactor/rhf-inputs/autocomplete-input';
+import { SWITCH_TYPE } from 'components/network/constants';
+import SelectInput from 'components/refactor/rhf-inputs/select-input';
+import { useWatch } from 'react-hook-form';
+import { useEffect, useState } from 'react';
+
+export const Connectivity = ({ id, index }) => {
```

Comment:
rename it to BusBarSectionConnection to avoid confusion with existing ConnectivityForm component ?